RockWeb/Themes/NewSpring/assets/Lava/UI/ad-unit.lava

```diff
@@ -14,7 +14,7 @@
         #{{ id }} {
             background-color: {{ backgroundcolor }};
         }
-    
+
```

Comment:
Same comment here as `get-permalink`. No changes were made, just spacing. Could get rid of this files changes.